test/specs/simulation.vehicles.spec.js

```diff
@@ -42,4 +42,17 @@ describe('randomBid()', () => {
       typeof bid.time_to_dropoff
     ).toBe('number');
   });
+
+  test('returns a price proportional to the total distance travelled', () => {
```

Comment:
I would describe as `returns a price that is influenced by the total distance travelled`.

It wouldn't be exactly proportional because different vehicles may have different rater per distance or time.

Reply:
@cg-cnu Feel free to merge after this text is changed